src/Xamarin.Android.Build.Tasks/Tasks/RemoveDirFixed.cs

```diff
@@ -76,7 +79,7 @@ public override bool RunTask ()
 			RemovedDirectories = temporaryRemovedDirectories.ToArray ();
 			Log.LogDebugTaskItems ("  RemovedDirectories: ", RemovedDirectories);
 
-			return true;
+			return !Log.HasLoggedErrors;
```

Comment:
I think this technically would return `false` if a previous task logged an error, but let the build keep going.  Not sure if that matters.

Reply:
Almost every task we return `!Log.HasLoggedErrors`, I think it is a good pattern.